From f0bbe2c6a0e2be2d64c1aabb8ea8ae67dd16ba5f Mon Sep 17 00:00:00 2001
From: Andrew Pinski <apinski@cavium.com>
Date: Tue, 30 Aug 2016 13:40:07 -0700
Subject: [PATCH 6/9] Fix non C++ build of gdb.

Older gdb does not build as C++ code, so need to use char instead of bool.

Signed-off-by: Andrew Pinski <apinski@cavium.com>
---
 gdb/aarch64-tdep.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdb/aarch64-tdep.h b/gdb/aarch64-tdep.h
index 7b1ce3e..b6b5567 100644
--- a/gdb/aarch64-tdep.h
+++ b/gdb/aarch64-tdep.h
@@ -96,7 +96,7 @@ struct gdbarch_tdep
   /* syscall record.  */
   int (*aarch64_syscall_record) (struct regcache *regcache, unsigned long svc_number);
   /* If this is ILP32 or LP64.  */
-  bool ilp32;
+  char ilp32;
 
 };
 
-- 
2.6.4

